gooddata-sdk/gooddata_sdk/catalog/organization/service.py

```diff
@@ -1,10 +1,30 @@
 # (C) 2022 GoodData Corporation
 from __future__ import annotations
 
+from typing import Optional
+
 from gooddata_sdk.catalog.catalog_service_base import CatalogServiceBase
+from gooddata_sdk.catalog.organization.entity_model.organization import CatalogOrganizationDocument
 from gooddata_sdk.client import GoodDataApiClient
 
 
 class CatalogOrganizationService(CatalogServiceBase):
     def __init__(self, api_client: GoodDataApiClient) -> None:
         super(CatalogOrganizationService, self).__init__(api_client)
+
+    def update_oidc_parameters(
```

Comment:
Unless it is too expensive, please, implement additional methods for updating "name" and "hostname", both are valid business cases.
Keep the new methods separate, they are different use cases.